src/routes/LandingPage/index.js

```diff
@@ -4,7 +4,7 @@ import { connect } from 'react-redux'
 import compose from 'lodash/fp/compose'
 
 import LOCATIONS_ENDPOINT from 'endpoints/location'
-import FilterableLocationSelector from 'components/LocationSelector/FilterableLocationSelector'
+import FilterableLocationSelector from 'components/FilterableLocationSelector/index'
```

Comment:
change `import FilterableLocationSelector from 'components/FilterableLocationSelector/index'` to `import FilterableLocationSelector from 'components/FilterableLocationSelector'`

Reply:
Done